gdal/frmts/prf/phprfdataset.cpp

```diff
@@ -608,11 +608,11 @@ GDALDataset* PhPrfDataset::Open( GDALOpenInfo* poOpenInfo )
             abDemMetadataOk[2] && abDemMetadataOk[3] )
         {
             adfGeoTrans[0] = adfDemMetadata[0];
-            adfGeoTrans[1] = (adfDemMetadata[1] - adfDemMetadata[0])/nSizeX;
+            adfGeoTrans[1] = (adfDemMetadata[1] - adfDemMetadata[0])/(nSizeX - 1);
```

Comment:
is there a check somewhere that nSizeX != 1 and nSizeY != 1 ?
And generally when you do this, it means that the native geolocation is given with respect to pixel centre. So to go to GDAL convention of pixel-corner, you have to apply a half-pixel offset to gt[0], gt[3]